static/_redirects

```diff
@@ -262,7 +262,7 @@
 /docs/tasks/administer-cluster/quota-memory-cpu-namespace/     /docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/ 301
 /docs/tasks/administer-cluster/quota-pod-namespace/     /docs/tasks/administer-cluster/manage-resources/quota-pod-namespace/ 301
 /docs/tasks/administer-cluster/reserve-compute-resources/out-of-resource.md     /docs/tasks/administer-cluster/out-of-resource/ 301
-/docs/tasks/administer-cluster/out-of-resource/     /docs/concepts/scheduling-eviction/pod-eviction/ 301
+/docs/tasks/administer-cluster/out-of-resource/     /docs/concepts/scheduling-eviction/node-pressure-eviction/ 301
```

Comment:
Hi @shannonxtreme .
I haven't checked, but what happens with the `pod-eviction` page?

Reply:
In #27739 we made new pages for types of evictions, moving content from out-of-resource (now deleted). It was originally one new page (pod-eviction) that I made, but we later split it up into two and changed the page names to match. 

I had set up redirects from the out-of-resource page to pod-eviction but forgot to update one of the redirects to point to the renamed page (Node-pressure eviction). 

Definitely not confusing at all 😅

Reply:
OK. I am wondering if there was a `pod-eviction` page that still needs to be redirected or if this page was never created.
https://kubernetes.io/docs/concepts/scheduling-eviction/pod-eviction

Reply:
We never created it, ended up changing the name before we merged, so nothing to redirect I think